saleor/graphql/order/tests/mutations/test_draft_order_create.py

```diff
@@ -1417,3 +1417,63 @@ def test_draft_order_create_with_non_unique_external_reference(
     assert error["field"] == "externalReference"
     assert error["code"] == OrderErrorCode.UNIQUE.name
     assert error["message"] == "Order with this External reference already exists."
+
+
+def test_draft_order_create_with_custom_price_in_order_line(
```

Comment:
We should add the similar test for unconfirmed orders. 
We should add tests with `force_new_line=True` for unconfirmed and draft orders.

Reply:
https://github.com/saleor/saleor/pull/12855/files#diff-1de8a795a5a985ab55d370dbaa4f569ba1f15615bdd0195426de4d4d36c5290bR752